optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessor.java

```diff
@@ -132,6 +134,33 @@ public String createResponseContent() {
         return responseContent.toString();
     }
 
+    public OptOutResults getOptOutResults() {
```

Comment:
can you please create new unit test for this method?

Reply:
Added a test. I think it covers what it should, but please double check for me :)